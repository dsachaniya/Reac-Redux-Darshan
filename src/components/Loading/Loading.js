import React from 'react';

/**
 * Loading
 * @returns {Node}
 */
const Loading = () => {
  return <div>Loading...</div>;
};

export default Loading;
