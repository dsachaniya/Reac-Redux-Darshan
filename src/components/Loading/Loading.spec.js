import React from 'react';
import { shallow } from 'enzyme';
import Loading from './Loading';

describe('<Loading />', () => {
  it('Rendering', () => {
    const wrapper = shallow(<Loading />);
    // 1. Snapshot
    expect(wrapper).toMatchSnapshot();

    // 2. Check For the <div>Loading...</div>
    const componentToTest = wrapper.find('div');
    expect(componentToTest).toHaveLength(1);
    expect(componentToTest.text()).toEqual('Loading...');
  });
});
