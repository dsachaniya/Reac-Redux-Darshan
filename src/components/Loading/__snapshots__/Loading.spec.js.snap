// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> Rendering 1`] = `
<div>
  Loading...
</div>
`;
